scripts/build-native.js

```diff
@@ -26,7 +28,8 @@ async function build() {
     console.log(`Building ${pkg}...`);
     await new Promise((resolve, reject) => {
       let args = [
-        (wasm ? 'wasm:' : '') + (release ? 'build-release' : 'build'),
+        (wasm ? 'wasm:' : '') +
```

Comment:
can we split this out into an if statement? Nested ternaries are hard to read

Reply:
Haha this is on me. Nik said the same thing - I agreed but was feeling lazy.